FireRender.Maya.Src/FireRenderToonMaterial.cpp

```diff
@@ -168,6 +171,10 @@ MStatus FireMaya::ToonMaterial::initialize()
 	MAKE_INPUT(nAttr);
 	nAttr.setConnectable(false);
 
+	Attribute::enableMidColorAsAlbedo = nAttr.create("enableMidColorAsAlbedo", "emcaa", MFnNumericData::kBoolean, 0);
```

Comment:
default value "false" is correct, right?

Reply:
yes